pyglider/slocum.py

```diff
@@ -807,9 +811,35 @@ def binary_to_timeseries(indir, cachedir, outdir, deploymentyaml, *,
     outdir : string
         Directory to put the merged timeseries files.
 
-    deploymentyaml : str
+    deploymentyaml : string
```

Comment:
```suggestion
    deploymentyaml : str
```

The python type is `str`